src/Data/Recruit.ChangeScripts/010_vacancy_backfill_closureReason.js

```diff
@@ -0,0 +1,48 @@
+{
+    const sleepTime = 500;
+    const autoClosure = "Auto";
+    const manualClosure = "Manual";
+
+    function setClosureReason(vac, closureReason) {
+        print(`updating ${vac.vacancyReference}`);
+        db.vacancies.updateOne({ "_id": vac._id }, { $set: { "closureReason": closureReason} });
```

Comment:
This is potentially going to send off 100 requests in a fraction of a second. Not sure if this might cause a `request rate is large`. Have you considered using an `updateMany`

Reply:
I did few quick tests (1500 updates back to back without any halts), there weren't any errors
There are 700+ vacancies on TEST, if it is a problem it will show up on TEST. 
I had tried UpdateMany before (for some other update) and it had failed with request rates.